README-dev.md

```diff
@@ -23,6 +23,9 @@ To see the test running in a web browser, go to `http://localhost:8000/`.
 
 ## Run tests
 
+If you are updating the bindings, you might need to clean first your
+environment using `npm run clean`.
```

Comment:
```suggestion
If you are have recently changed branches, you might need to clean first your
environment using `npm run clean`.
```
how about this? it's more general and doesn't have to do with updating the bindings. Actually, `npm run bindings` does run `npm run clean`

Reply:
Your suggestion makes more sense as it is what certainly happened to me.